www/caddy/src/opnsense/service/templates/OPNsense/Caddy/Caddyfile

```diff
@@ -277,6 +277,13 @@
     {% if autoHttpsValue %}
     auto_https {{ autoHttpsValue }}
     {% endif %}
+    {# 
+    # Important: Grace Period influences how fast the server can finish reloads with open connections, by forcing termination.
+    # Default of Caddy is to wait for all connections to close before allowing reload, meaning the higher the value, the longer applies take.
+    #}
+    {% if generalSettings.GracePeriod %}
+    grace_period {{ generalSettings.GracePeriod }}s
+    {% endif %}
```

Comment:
Since it's required and has a default:

```suggestion
    grace_period {{ generalSettings.GracePeriod }}s
```

Reply:
Sounds good, I just always want to make extra sure. xD